src/components/IconButtonBar/_mixins.scss

```diff
@@ -36,13 +36,12 @@
         width: $icon-button-bar__sizing__dimensions;
       }
     }
-  }
-
-  &__overflow-menu-options {
-    box-shadow: unset;
 
-    &.#{$prefix}--overflow-menu-options::after {
-      height: 0;
+    &__overflow-menu-options--#{$icon-button-bar__size}.bx--overflow-menu--flip {
+      &.#{$prefix}--overflow-menu-options[data-floating-menu-direction='top']::after,
+      &.#{$prefix}--overflow-menu-options[data-floating-menu-direction='bottom']::after {
```

Comment:
The style rules being overridden are applied to: `.#{$prefix}--overflow-menu-options[data-floating-menu-direction='top']::after` & `.#{$prefix}--overflow-menu-options[data-floating-menu-direction='bottom']::after`, so that's why this rule is so specific